tests/examples/test_example_multi.py

```diff
@@ -49,6 +51,7 @@ def test_example_multi(self):
         cfg.train_for_env_steps = 100000
         cfg.num_workers = 8
         cfg.batch_size = 512
+        cfg.batched_sampling = True
```

Comment:
Why did you change this to batched sampling?

I think for multi-agent we actually want to test non-batched version because some of the features (multi-policy training and inactive agents are actually only supported in non-batched mode)

Reply:
Sure, I'll change back to non-batched